.github/workflows/create-quantic-package.yml

```diff
@@ -65,8 +65,8 @@ jobs:
       - name: Create package version
         working-directory: ./packages/quantic
         run: |
-          sfdx force:package:version:create --package $PACKAGE_ID --wait 30 -x --json >> package-version-result.json
+          sfdx force:package:version:create --package $PACKAGE_ID --codecoverage --installationkeybypass --wait 30 --json &> package-version-result.json
```

Comment:
:thinking: Why are you adding `--codecoverage`? According to [Salesforce documentation](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_package.htm#cli_reference_force_package_version_create), adding it promotes the package.

Reply:
Yes you're right it shouldn't be there. I just removed it.